vmdb/app/models/mixins/web_server_worker_mixin.rb

```diff
@@ -69,7 +81,7 @@ def self.sync_workers
             port = self.reserve_port(ports)
             $log.info("MIQ(#{self.name}.sync_workers) Reserved port=#{port}, Current ports in use: #{ports.inspect}")
             ports << port
-            w = self.start_worker(:uri => "http://0.0.0.0:#{port}")
+            w = self.start_worker(:uri => build_uri(port))
```

Comment:
Yes @kbrock... thanks, that's not obvious.  I'll add a comment.  Specifically, when we need to run in IPv6 only environments, we'll need to set BINDING_ADDRESS = "::1".  I already didn't like building the URI in-line with string interpolation so I handled both in one change.